src/components/endpoints/SendTransaction.vue

```diff
@@ -151,6 +161,96 @@ const sampleData = computed<{ [key: string]: TransactionSelect[] }>(() => {
         )}0000000000000000000000000000000000000000000000056bc75e2d63100000`,
       },
     ],
+    LSP7: [
+      {
+        label: '💰 Transfer: 1 token A (divisible) to EoA',
+        to: '0x314E7a56B08AF8E729612930dBAd70BB5A3575D9',
+        amount: 0,
+        hasData: true,
+        data: `760d9bba000000000000000000000000${currentUP.substring(
+          2
+        )}000000000000000000000000${sampleEoA.substring(
+          2
+        )}0000000000000000000000000000000000000000000000000de0b6b3a7640000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000000`,
+      },
+      {
+        label: '💰 Transfer: 1 token A (divisible) to EoA (without force)',
+        to: '0x314E7a56B08AF8E729612930dBAd70BB5A3575D9',
```

Comment:
I suggest to put this in a variable:

const tokenScAddress = xxx

Will be easier to understand what it is, and it is also used a couple of times below.